plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/HiveS3Config.java

```diff
@@ -354,6 +355,19 @@ public HiveS3Config setS3ConnectTimeout(Duration s3ConnectTimeout)
         return this;
     }
 
+    public Duration getS3ConnectTtl()
+    {
+        return s3ConnectTtl;
+    }
+
+    @Config("hive.s3.connect-ttl")
+    @ConfigDescription("TCP connect ttl in the client side, which affects connection reusage")
```

Comment:
nit: Uppercase "ttl".

Reply:
> nit: Uppercase "ttl".

Thank you for your advice above, i have improved the aspects you mentioned above, please review again when you have time~